Examples/PlatformExample.cs

```diff
@@ -27,10 +27,10 @@ public override string ToString()
             }
         }
 
-        public static async Task Main(string[] args)
+        public static async Task Main()
         {
-            var influxDB = InfluxDBClientFactory.Create("http://localhost:9999",
-                "my-user", "my-password".ToCharArray());
+            var influxDB = new InfluxDBClient("http://localhost:9999",
```

Comment:
Please refactor to `using(...)`:

```suggestion
            using (var influxDB = new InfluxDBClient("http://localhost:9999",
                       "my-user", "my-password"))
            {
```